From 42f9040301745e692e457580616cdcef5a0f8f81 Mon Sep 17 00:00:00 2001
From: azdavis <ariel.z.davis@icloud.com>
Date: Sat, 4 Jun 2016 23:55:09 -0400
Subject: [PATCH] coreutils-remove-g-prefix

---
 Formula/coreutils.rb | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/Formula/coreutils.rb b/Formula/coreutils.rb
index e2af40d..e9e539b 100644
--- a/Formula/coreutils.rb
+++ b/Formula/coreutils.rb
@@ -46 +45,0 @@ class Coreutils < Formula
-      --program-prefix=g
@@ -51,13 +49,0 @@ class Coreutils < Formula
-
-    # Symlink all commands into libexec/gnubin without the 'g' prefix
-    coreutils_filenames(bin).each do |cmd|
-      (libexec/"gnubin").install_symlink bin/"g#{cmd}" => cmd
-    end
-    # Symlink all man(1) pages into libexec/gnuman without the 'g' prefix
-    coreutils_filenames(man1).each do |cmd|
-      (libexec/"gnuman"/"man1").install_symlink man1/"g#{cmd}" => cmd
-    end
-
-    # Symlink non-conflicting binaries
-    bin.install_symlink "grealpath" => "realpath"
-    man1.install_symlink "grealpath.1" => "realpath.1"
-- 
2.8.3

