--- a/sublime-text-3/Packages/User/Preferences.sublime-settings
+++ b/sublime-text-3/Packages/User/Preferences.sublime-settings
@@ -9,3 +9,3 @@
 	"close_windows_when_empty": false,
-	"color_scheme": "Packages/Color Scheme - Default/Sixteen.sublime-color-scheme",
+	"color_scheme": "Packages/Color Scheme - Default/Monokai.sublime-color-scheme",
 	"copy_with_empty_selection": false,
@@ -50,2 +50,3 @@
 	[
+		"gray_antialias",
 		"no_bold",
@@ -71,2 +72,3 @@
 	[
+		"gray_antialias"
 	],
--- a/vscode/user-data/User/settings.json
+++ b/vscode/user-data/User/settings.json
@@ -97,3 +97,3 @@
     "window.restoreWindows": "none",
-    "workbench.colorTheme": "Default Light+",
+    "workbench.colorTheme": "Default Dark+",
     "workbench.editor.restoreViewState": false,
