--- a/git/config
+++ b/git/config
@@ -15,3 +15,3 @@
   eol = lf
-  pager = delta --dark
+  pager = delta --light
   quotePath = false
@@ -27,3 +27,3 @@
 [interactive]
-  diffFilter = delta --color-only --dark
+  diffFilter = delta --color-only --light
 [log]
--- a/vscode/settings.json
+++ b/vscode/settings.json
@@ -51,3 +51,3 @@
   "window.restoreWindows": "none",
-  "workbench.colorTheme": "Default Dark+",
+  "workbench.colorTheme": "Default Light+",
   "workbench.editor.enablePreview": false,
