From 6aefe99587cc30b1861d610919cf244cfdde785d Mon Sep 17 00:00:00 2001
From: azdavis <ariel.z.davis@icloud.com>
Date: Sat, 4 Jun 2016 23:52:43 -0400
Subject: [PATCH] presentation-styles

---
 subl/Preferences.sublime-settings | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/subl/Preferences.sublime-settings b/subl/Preferences.sublime-settings
index 1d72761..5577025 100644
--- a/subl/Preferences.sublime-settings
+++ b/subl/Preferences.sublime-settings
@@ -75 +75 @@
-    "font_size": 16,
+    "font_size": 24,
@@ -112 +111,0 @@
-        79
@@ -118 +117 @@
-    "spell_check": true,
+    "spell_check": false,
-- 
2.8.3

