diff --git a/sublime-text-3/Packages/User/theme/theme.tmTheme b/sublime-text-3/Packages/User/theme/theme.tmTheme
index 39d0990..328ad49 100644
--- a/sublime-text-3/Packages/User/theme/theme.tmTheme
+++ b/sublime-text-3/Packages/User/theme/theme.tmTheme
@@ -56,3 +56,3 @@
                 <key>scope</key>
-                <string>entity.name.tag, keyword, markup.deleted, markup.heading punctuation, meta.selector, meta.tag, storage, support.function</string>
+                <string>markup.deleted</string>
                 <key>settings</key>
@@ -65,3 +65,3 @@
                 <key>scope</key>
-                <string>variable.language, constant.language.this</string>
+                <string>-</string>
                 <key>settings</key>
@@ -83,3 +83,3 @@
                 <key>scope</key>
-                <string>entity.name.type, entity.other.attribute-name, markup.inserted, meta.constant, support.class, variable.other.constant</string>
+                <string>markup.inserted</string>
                 <key>settings</key>
@@ -92,3 +92,3 @@
                 <key>scope</key>
-                <string>entity.name.function, markup.underline.link, meta.diff</string>
+                <string>meta.diff</string>
                 <key>settings</key>
