diff --git a/subl/Preferences.sublime-settings b/subl/Preferences.sublime-settings
index eb85b83..ffc581d 100644
--- a/subl/Preferences.sublime-settings
+++ b/subl/Preferences.sublime-settings
@@ -21,3 +21,2 @@
     [
-        "gray_antialias",
         "no_bold",
diff --git a/subl/theme/Widget - theme.tmTheme b/subl/theme/Widget - theme.tmTheme
index 4581266..dbfb420 100644
--- a/subl/theme/Widget - theme.tmTheme
+++ b/subl/theme/Widget - theme.tmTheme
@@ -10,9 +10,9 @@
                     <key>background</key>
-                    <string>#111</string>
+                    <string>#ddd</string>
                     <key>caret</key>
-                    <string>#ddd</string>
+                    <string>#222</string>
                     <key>foreground</key>
-                    <string>#ddd</string>
+                    <string>#222</string>
                     <key>selection</key>
-                    <string>#333</string>
+                    <string>#ccc</string>
                 </dict>
@@ -36,3 +36,3 @@
                     <key>foreground</key>
-                    <string>#d99</string>
+                    <string>#900</string>
                 </dict>
@@ -45,3 +45,3 @@
                     <key>foreground</key>
-                    <string>#9d9</string>
+                    <string>#090</string>
                 </dict>
@@ -54,3 +54,3 @@
                     <key>foreground</key>
-                    <string>#9dd</string>
+                    <string>#099</string>
                 </dict>
@@ -63,3 +63,3 @@
                     <key>foreground</key>
-                    <string>#99d</string>
+                    <string>#009</string>
                 </dict>
diff --git a/subl/theme/theme.sublime-theme b/subl/theme/theme.sublime-theme
index dcb24fa..a1008b8 100644
--- a/subl/theme/theme.sublime-theme
+++ b/subl/theme/theme.sublime-theme
@@ -4,3 +4,3 @@
         "layer0.texture": "",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.inner_margin": 0,
@@ -20,3 +20,3 @@
         "layer0.texture": "",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.inner_margin": 5,
@@ -33,3 +33,3 @@
         "layer0.texture": "",
-        "layer0.tint": [51, 51, 51]
+        "layer0.tint": [213, 213, 213]
     },
@@ -98,3 +98,3 @@
         "parents": [{"class": "tab_control", "attributes": ["selected"]}],
-        "fg": [221, 221, 221]
+        "fg": [17, 17, 17]
     },
@@ -126,3 +126,3 @@
         "layer0.texture": "",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.opacity": 1,
@@ -135,3 +135,3 @@
         "layer0.texture": "",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.inner_margin": 0,
@@ -142,3 +142,3 @@
         "layer0.texture": "",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.inner_margin": 0,
@@ -149,3 +149,3 @@
         "layer0.texture": "",
-        "layer0.tint": [51, 51, 51],
+        "layer0.tint": [213, 213, 213],
         "layer0.opacity": 1,
@@ -158,3 +158,3 @@
         "attributes": ["horizontal"],
-        "layer0.tint": [51, 51, 51],
+        "layer0.tint": [213, 213, 213],
         "layer0.inner_margin": 0,
@@ -176,3 +176,3 @@
         "settings": ["overlay_scroll_bars"],
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.inner_margin": [0, 5],
@@ -220,3 +220,3 @@
         "class": "sheet_container_control",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.opacity": 1
@@ -226,3 +226,3 @@
         "border_size": 1,
-        "border_color": [34, 34, 34]
+        "border_color": [204, 204, 204]
     },
@@ -230,3 +230,3 @@
         "class": "minimap_control",
-        "viewport_color": [221, 221, 221, 30]
+        "viewport_color": [170, 170, 170, 30]
     },
@@ -256,3 +256,3 @@
         "class": "tool_tip_label_control",
-        "color": [221, 221, 221]
+        "color": [17, 17, 17]
     },
@@ -261,3 +261,3 @@
         "layer0.texture": "",
-        "layer0.tint": [17, 17, 17],
+        "layer0.tint": [221, 221, 221],
         "layer0.opacity": 1,
@@ -277,3 +277,3 @@
         "layer0.opacity": 1,
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.inner_margin": [1, 5, 2, 1],
@@ -291,3 +291,3 @@
         "class": "tree_row",
-        "layer0.tint": [51, 51, 51],
+        "layer0.tint": [213, 213, 213],
         "layer0.opacity": 0,
@@ -322,3 +322,3 @@
         "parents": [{"class": "tree_row", "attributes": ["selected"]}],
-        "color": [221, 221, 221]
+        "color": [17, 17, 17]
     },
@@ -353,3 +353,3 @@
         "layer0.opacity": 1,
-        "layer0.tint": [17, 17, 17],
+        "layer0.tint": [221, 221, 221],
         "layer0.inner_margin": 8
@@ -358,3 +358,3 @@
         "class": "text_line_control",
-        "layer0.tint": [17, 17, 17],
+        "layer0.tint": [221, 221, 221],
         "layer0.opacity": 1,
@@ -366,3 +366,3 @@
         "layer0.opacity": 1,
-        "layer0.tint": [17, 17, 17],
+        "layer0.tint": [221, 221, 221],
         "content_margin": 0
@@ -372,3 +372,3 @@
         "layer0.opacity": 1,
-        "layer1.tint": [34, 34, 34],
+        "layer1.tint": [204, 204, 204],
         "layer1.inner_margin": 0,
@@ -380,3 +380,3 @@
         "row_padding": 8,
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.opacity": 1,
@@ -386,3 +386,3 @@
         "class": "quick_panel_row",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.inner_margin": 8,
@@ -393,3 +393,3 @@
         "attributes": ["selected"],
-        "layer0.tint": [51, 51, 51]
+        "layer0.tint": [213, 213, 213]
     },
@@ -400,3 +400,3 @@
         "selected_fg": [170, 170, 170],
-        "selected_match_fg": [221, 221, 221]
+        "selected_match_fg": [17, 17, 17]
     },
@@ -416,3 +416,3 @@
         "class": "mini_quick_panel_row",
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.opacity": 1
@@ -422,3 +422,3 @@
         "attributes": ["selected"],
-        "layer0.tint": [51, 51, 51]
+        "layer0.tint": [213, 213, 213]
     },
@@ -427,3 +427,3 @@
         "content_margin": 0,
-        "layer0.tint": [34, 34, 34],
+        "layer0.tint": [204, 204, 204],
         "layer0.opacity": 1
@@ -443,3 +443,3 @@
         "class": "table_row",
-        "layer0.tint": [51, 51, 51],
+        "layer0.tint": [213, 213, 213],
         "layer0.opacity": 0,
@@ -477,3 +477,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -494,3 +494,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -511,3 +511,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -528,3 +528,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -545,3 +545,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -562,3 +562,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -579,3 +579,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -596,3 +596,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -613,3 +613,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
@@ -630,3 +630,3 @@
         "parents": [{"class": "icon_button_control", "attributes": ["selected"]}],
-        "layer0.tint": [221, 221, 221]
+        "layer0.tint": [17, 17, 17]
     },
diff --git a/subl/theme/theme.tmTheme b/subl/theme/theme.tmTheme
index fdff98c..2091081 100644
--- a/subl/theme/theme.tmTheme
+++ b/subl/theme/theme.tmTheme
@@ -10,15 +10,15 @@
                     <key>background</key>
-                    <string>#000</string>
+                    <string>#eee</string>
                     <key>caret</key>
-                    <string>#ddd</string>
+                    <string>#222</string>
                     <key>foreground</key>
-                    <string>#ddd</string>
+                    <string>#222</string>
                     <key>selection</key>
-                    <string>#333</string>
+                    <string>#ccc</string>
                     <key>lineHighlight</key>
-                    <string>#222</string>
+                    <string>#ccc</string>
                     <key>gutterForeground</key>
-                    <string>#555</string>
+                    <string>#aaa</string>
                     <key>guide</key>
-                    <string>#555</string>
+                    <string>#aaa</string>
                 </dict>
@@ -31,3 +31,3 @@
                     <key>foreground</key>
-                    <string>#ddd</string>
+                    <string>#222</string>
                 </dict>
@@ -40,3 +40,3 @@
                     <key>foreground</key>
-                    <string>#999</string>
+                    <string>#555</string>
                 </dict>
@@ -60,3 +60,3 @@
                     <key>foreground</key>
-                    <string>#d99</string>
+                    <string>#933</string>
                 </dict>
@@ -69,3 +69,3 @@
                     <key>foreground</key>
-                    <string>#d9d</string>
+                    <string>#939</string>
                 </dict>
@@ -78,3 +78,3 @@
                     <key>foreground</key>
-                    <string>#dd9</string>
+                    <string>#993</string>
                 </dict>
@@ -87,3 +87,3 @@
                     <key>foreground</key>
-                    <string>#9d9</string>
+                    <string>#393</string>
                 </dict>
@@ -96,3 +96,3 @@
                     <key>foreground</key>
-                    <string>#9dd</string>
+                    <string>#399</string>
                 </dict>
@@ -105,3 +105,3 @@
                     <key>foreground</key>
-                    <string>#99d</string>
+                    <string>#339</string>
                 </dict>
