diff --git a/Formula/coreutils.rb b/Formula/coreutils.rb
index d01bd5c..8b085da 100644
--- a/Formula/coreutils.rb
+++ b/Formula/coreutils.rb
@@ -44,24 +44,10 @@ class Coreutils < Formula
     system "./bootstrap" if build.head?
     args = %W[
       --prefix=#{prefix}
-      --program-prefix=g
     ]
     args << "--without-gmp" if build.without? "gmp"
     system "./configure", *args
     system "make", "install"
-
-    # Symlink all commands into libexec/gnubin without the 'g' prefix
-    coreutils_filenames(bin).each do |cmd|
-      (libexec/"gnubin").install_symlink bin/"g#{cmd}" => cmd
-    end
-    # Symlink all man(1) pages into libexec/gnuman without the 'g' prefix
-    coreutils_filenames(man1).each do |cmd|
-      (libexec/"gnuman"/"man1").install_symlink man1/"g#{cmd}" => cmd
-    end
-
-    # Symlink non-conflicting binaries
-    bin.install_symlink "grealpath" => "realpath"
-    man1.install_symlink "grealpath.1" => "realpath.1"
   end

   def caveats; <<-EOS.undent
