diff --git a/Formula/gnupg.rb b/Formula/gnupg.rb
index 23b7e45..5e007ba 100644
--- a/Formula/gnupg.rb
+++ b/Formula/gnupg.rb
@@ -24,3 +24,2 @@ class Gnupg < Formula
       --disable-asm
-      --program-suffix=1
     ]
@@ -36,11 +35,2 @@ class Gnupg < Formula
     system "make", "install"
-
-    # https://lists.gnupg.org/pipermail/gnupg-devel/2016-August/031533.html
-    inreplace bin/"gpg-zip1", "GPG=gpg", "GPG=gpg1"
-
-    # Although gpg2 support should be pretty universal these days
-    # keep vanilla `gpg` executables available, at least for now.
-    %w[gpg-zip1 gpg1 gpgsplit1 gpgv1].each do |cmd|
-      (libexec/"gpgbin").install_symlink bin/cmd => cmd.to_s.sub(/1/, "")
-    end
   end
